mattumland/next-graphcms-shows
# Add spotify link to show page

## What issue does this solve?
https://codevalapp.github.io/next-graphcms-shows/issues/6/

## Background
Some artists have spotify links. Let's show 'em. 

## Manual testing instructions
1. Boot up dev server
2. Go to `shows`
3. Click on `Made of Rain` and `A Return to Opera`, confirm that spotify links appear.
